src/main/java/org/broadinstitute/hellbender/tools/walkers/contamination/CalculateContamination.java

```diff
@@ -18,20 +19,35 @@
 import java.util.stream.IntStream;
 
 /**
- * Given pileup data from {@link GetPileupSummaries} calculate the fraction of reads coming from exogenous human DNA.
+ * Given pileup data from {@link GetPileupSummaries}, calculates the fraction of reads coming from cross-sample contamination.
  *
- * This tool is run as follows:
+ * <p>
+ *     The resulting contamination table is used with either {@link org.broadinstitute.hellbender.tools.walkers.mutect.Mutect2}
```

Comment:
If I `gatk-launch Mutect2`, I see that Mutect2 also has 
```
--contamination_fraction_per_sample_file,-contaminationFile:File
                              Tab-separated File containing fraction of contamination in sequencing data (per sample) to
                              aggressively remove. Format should be "<SampleID><TAB><Contamination>" (Contamination is
                              double) per line; No header.  Default value: null. 
```

But I can remove this if you think that it is better to only filter contamination using CalculateContamination.